metomi/rose/config_editor/__init__.py

```diff
@@ -723,7 +723,7 @@
 META_PROP_WIDGET_SUB_NS = "widget[rose-config-edit:sub-ns]"
 
 # Miscellaneous
-COPYRIGHT = """Copyright (C) 2012-2024 British Crown (Met Office) & Contributors.
+COPYRIGHT = """Crown Copyright (C) 2012-2024 (Met Office) & Contributors.
```

Comment:
```suggestion
COPYRIGHT = """Copyright (C) British Crown (Met Office) & Contributors.
```

Reply:
@oliver-sanders, please check that copyright notices in the GUI code match the new format.